packages/serverless-contracts/src/contracts/eventBridge/types/putEvent.ts

```diff
@@ -1,14 +1,19 @@
 import { EventBridgeClient } from '@aws-sdk/client-eventbridge';
+import { PutEventsResultEntry } from '@aws-sdk/client-eventbridge/dist-types/models/models_0';
```

Comment:
Is this not exported from a better path, even in more recent versions?

Reply:
The auto import of my IDE is fucked up. I fixed it